CombineExt.xcodeproj/project.pbxproj

```diff
@@ -497,6 +498,7 @@
 		OBJ_20 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
This will add version 1 into bundle. Does bundle actually should be versioned to be inline with whatever release it is? So for the current master 1.3.0?

Reply:
Are we talking about current version (1.2.0) or a future version?